### PR TITLE
will modify the failing tests

### PR DESCRIPTION
**Pull request by bradsawadye**
Some tests were failing on node version 10.15.1 This is a fix for the tests

OHM-751 OHM-748

### CODE REVIEW

File: test/unit/auditingTest.js [hunk 1]
```diff
@@ -301,7 +301,7 @@ describe('Auditing', () => {
 
     after(async () => {
       config.auditing.auditEvents = JSON.parse(_restore)
-      await Promise.all(servers.map(s => s.close()))
+      await servers.map(s => s.close())
```
Comment (bausmeier) [inline test/unit/auditingTest.js]: What is this line supposed to do now? Why the `.map` and `await`?

Comment (bradsawadye) [reply test/unit/auditingTest.js]: Hey Brett, so I am still investigating, havent made much headway as to why they fail. The tests only pass when the close function is not in a promise. The program times out before the close is event is fired. I will push something that's a bit cleaner just now

File: test/integration/tcpIntegrationTests.js [hunk 2]
```diff
@@ -228,6 +228,7 @@ describe('TCP/TLS/MLLP Integration Tests', () => {
     await Promise.all([
       TransactionModel.deleteMany({})
     ])
+    mockServer.close()
```
Comment (bausmeier) [inline test/integration/tcpIntegrationTests.js]: We're still not waiting for this to complete or handling any errors which might occur during close.

File: test/unit/auditingTest.js [hunk 3]
```diff
@@ -301,7 +301,7 @@ describe('Auditing', () => {
 
     after(async () => {
       config.auditing.auditEvents = JSON.parse(_restore)
-      await servers.map(s => s.close())
+      servers.map(s => s.close())
```
Comment (bausmeier) [inline test/unit/auditingTest.js]: We're still not waiting for the servers to close or handling any errors that may happen.

File: test/unit/auditingTest.js (hunk 1 above)
Comment (bradsawadye) [reply test/unit/auditingTest.js]: > What is this line supposed to do now? Why the `.map` and `await`?

the line is for closing the servers that are in an array

**Pull request closed without merging**